sdk_contrib/fastify/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "aws-xray-sdk-fastify",
+  "version": "3.3.6",
+  "description": "AWS X-Ray plugin for Fastify",
+  "author": "Amazon Web Services",
+  "contributors": [
+    "Jorge Vargas <jorgevargasla@gmail.com>"
+  ],
+  "main": "lib/index.js",
+  "files": [
+    "lib"
+  ],
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
+    "test": "mocha --recursive --exit ./test/ -R spec && tsd",
+    "test-d": "tsd",
+    "sample": "node sample"
+  },
+  "engines": {
+    "node": ">= 12.x"
+  },
+  "peerDependencies": {
+    "aws-xray-sdk-core": "^3.3.6",
+    "fastify": "^3.29.0"
+  },
+  "keywords": [
+    "amazon",
+    "api",
+    "aws",
+    "fastify",
+    "xray",
+    "x-ray",
+    "x ray"
+  ],
+  "directories": {
+    "lib": "lib"
+  },
+  "license": "Apache-2.0",
+  "repository": "https://github.com/aws/aws-xray-sdk-node/tree/sdk_contrib/fastify",
+  "devDependencies": {
```

Comment:
Shouldn't be any dev dependencies, they should be hoisted

Reply:
Working on this change

Reply:
I have tried `npx lerna run test --scope=aws-xray-sdk-fastify` and now it seems to pass all the tests.